test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/rejectedgoodsscheduled/CheckYourAnswersControllerSpec.scala

```diff
@@ -59,13 +62,15 @@ class CheckYourAnswersControllerSpec
 
   def mockSubmitClaim(submitClaimRequest: RejectedGoodsScheduledClaimConnector.Request)(
     response: Future[RejectedGoodsScheduledClaimConnector.Response]
-  ) =
+  ): CallHandler2[RejectedGoodsScheduledClaimConnector.Request, HeaderCarrier, Future[
+    RejectedGoodsScheduledClaimConnector.Response
+  ]] =
     (mockConnector
       .submitClaim(_: RejectedGoodsScheduledClaimConnector.Request)(_: HeaderCarrier))
       .expects(submitClaimRequest, *)
       .returning(response)
 
-  def mockWipeOutCall() =
+  def mockWipeOutCall(): CallHandler1[HeaderCarrier, Future[Unit]] =
```

Comment:
I'm not sure if adding this kind of return type to the definition helps?

Reply:
It doesn't do any harm to add them. Also it's good practice to add type annotations to public members. At least IntelliJ shows some warnings/suggestions for this.